src/crypto/crypto_ml_dsa.cc

```diff
@@ -0,0 +1,73 @@
+#include "crypto/crypto_ml_dsa.h"
+#include "crypto/crypto_util.h"
+#include "env-inl.h"
+#include "string_bytes.h"
+#include "v8.h"
+
+namespace node {
+
+using ncrypto::DataPointer;
+using v8::Local;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+namespace crypto {
+
+#if OPENSSL_VERSION_MAJOR >= 3 && OPENSSL_VERSION_MINOR >= 5
+bool ExportJwkMlDsaKey(Environment* env,
```

Comment:
ML-DSA JWK keys have the following format

```js
{
  "kty": "AKP", // required
  "alg": "ML-DSA-XX", // required
  "pub": "<base64url encoded raw public key>", // required
  "priv": "<base64url encoded raw seed>", // required for private keys, prohibited for public keys
}

Reply:
it would be helpful to include this in a comment in the code :-)